subjects/03-components/exercise.js

```diff
@@ -30,18 +30,49 @@ styles.panel = {
   padding: 10
 }
 
+
+const DATA = [
+  { id: 1, name: 'USA', description: 'Land of the Free, Home of the brave' },
+  { id: 2, name: 'Brazil', description: 'Sunshine, beaches, and Carnival' },
+  { id: 3, name: 'Russia', description: 'World Cup 2018!' },
+  { id: 4, name: 'UK', description: 'Current Location' }
+]
+
 const Tabs = React.createClass({
+  propTypes: {
```

Comment:
I avoided introducing any country-specific naming to the `Tabs` component, but missed an opportunity to have `App` `map` over its data; `name` and `description` aren't ideal property names for tabs.